CHANGELOG.md

```diff
@@ -1,109 +1,167 @@
-# 1.0.0-rc1
-
-## November 6, 2022
+# 0.8.0-rc0
+
+## February 8, 2023
+
+0.8.0 is a breaking change that contains all updates since release 0.7.5. It
+extends the previously tagged `1.0.0-rc1` release because it is not clear when
+that release will be finalized. Many of the PRs in this release were previously
+included in the `1.0.0-rc1` release. The categorized list of PRs separates those
+that are new from those in the `1.0.0-rc1` release candidate.
+
+With this release, a new automated process publishes container
+images in the Hyperledger container image repository for this version of ACA-Py
+based on both Python 3.6 and 3.9. We recommend using Python 3.9 with ACA-Py.
```

Comment:
Additional information can be found here; [Container Images and Github Actions](https://github.com/hyperledger/aries-cloudagent-python/blob/main/ContainerImagesAndGithubActions.md)

The new images will be published here; https://github.com/orgs/hyperledger/packages?repo_name=aries-cloudagent-python.  There will be a link to the packages on the repository's main page under **Packages** providing a more direct link once the first images are published.

The images are based on `python slim-bullseye` images, and are built to support `linux/386` (`x86`), `linux/amd64` (`x64`), and `linux/arm64`.  There are two flavors of image built for each python version.  One containing the indy/aries shared libraries only, and one containing the indy/aries shared libraries and the indy-sdk.  The images containing the indy-sdk are labeled `indy`.